src/pages/home/sidebar/SidebarScreen.js

```diff
@@ -168,6 +169,11 @@ class SidebarScreen extends Component {
                                     text: this.props.translate('sidebarScreen.newGroup'),
                                     onSelected: () => Navigation.navigate(ROUTES.NEW_GROUP),
                                 },
+                                {
+                                    icon: Hashtag,
+                                    text: this.props.translate('sidebarScreen.newRoom'),
+                                    onSelected: () => Navigation.navigate(ROUTES.WORKSPACE_NEW_ROOM),
+                                },
```

Comment:
I know this wasn't part of the doc, but since this feature has been de-prioritized in the roadmap, perhaps we want to put it behind a beta for now? So we don't have a non-complete feature out there with our customers?

Reply:
Will open up a Web PR for setting up betas